changes/process_changes.py

```diff
@@ -5,14 +5,21 @@
 from __future__ import absolute_import, division, print_function
 
 import argparse as A
+from collections import OrderedDict
 from glob import glob
 import os
 import os.path as P
+import sys
 import yaml
 
 from funcy import memoize
 
 
+def print_err(*args, **kwargs):
```

Comment:
No doc, also a bit convoluted way to write it, took me a while to understand that it's the same as

```py
print(*args, file=sys.stderr, **kwargs)
```

Reply:
Right, I forgot to add it: will do, thanks.